.prettierrc

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Please remove this file. What is vscode not happy about ?

Reply:
this is the most up to date way to share prettier config across all related tooling 
https://prettier.io/docs/en/configuration.html
with this file, vscode prettier extension simply picking the correct configurations

Reply:
We included prettier as part of eslint with `eslint-config-prettier` and `eslint-plugin-prettier`. Configure your editor to apply eslint on save instead of prettier and you'll be ok :)

Reply:
In VScode, this is done through the eslint extension

Reply:
If we can have this file instead of our rules inside `.eslintrc` I'll merge that separate PR :) I just don't want our rules duplicated in two different files